### PR TITLE
docs(events): Adding content - INNO-2046

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

**Review by planctus — DISMISSED** [review]
image

Missing spaces after the comma in the links.

File: src/website/src/pages/ec/content-types/events/index.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Events
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  The event content type is organised and structured throughout multiple pages (
+  <a href="/ec/content-types/events/agenda/">agenda</a>,{' '}
```
Comment (kalinchernev) [inline src/website/src/pages/ec/content-types/events/index.mdx]: Native links are not wrong, but reload the page on browsing. To make the transition smoother, a `Link` component is provided. You can take it by `import { Paragraph, Link } from '@ecl/website-components'`. 

An example usage then will be 

```js
<Link to="/ec/components/forms/radio/usage/">radio buttons</Link>
```

**Review by kalinchernev — CHANGES_REQUESTED** [review]

File: src/website/src/pages/ec/content-types/events/index.mdx (hunk 1 above)
Comment (papegaill) [reply src/website/src/pages/ec/content-types/events/index.mdx]: @kalinchernev Thank you, but I think there is an issue using Link inside a Paragraph, It render correctly but throwing an error during push:
`[error] src/website/src/pages/ec/content-types/events/index.mdx: SyntaxError: Void elements do not have end tags "Link" (3:53)`

I think this was the same for src/website/src/pages/ec/content-types/policy/index.mdx

Comment (kalinchernev) [reply src/website/src/pages/ec/content-types/events/index.mdx]: It's possible to include components inside by wrapping them in `{}`:

```js
---
title: Events
---

import { Paragraph, Link } from '@ecl/website-components';

<Paragraph size="lead">
  The event content type is organised and structured throughout multiple pages (
  {<Link to="/ec/content-types/events/agenda/">agenda</Link>},{' '}
  {<Link to="/ec/content-types/events/detail/">details</Link>},{' '}
  {<Link to="/ec/content-types/events/speaker/">speaker</Link>}) in order to minimise
  the overload of information.
</Paragraph>
<Paragraph size="lead">
  The content patterns for each page were created according to user research,
  which indicated how content of interest can be either{' '}
  <strong>arranged or progressively disclosed</strong>, by{' '}
  <strong>encouraging further navigation to the auxiliary pages</strong>.
</Paragraph>
```

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**